bundles/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/message/PowermaxCommManager.java

```diff
@@ -95,10 +95,10 @@
     /**
      * Constructor for Serial Connection
      *
-     * @param sPort             the serial port name
-     * @param panelType         the panel type to be used when in standard mode
+     * @param sPort the serial port name
+     * @param panelType the panel type to be used when in standard mode
      * @param forceStandardMode true to force the standard mode rather than trying using the Powerlink mode
-     * @param autoSyncTime      true for automatic sync time
+     * @param autoSyncTime true for automatic sync time
```

Comment:
I don't know how to feel about changing all the formatting of the javadocs in this file. It looks like the alignments were very intentional for human readability.

Reply:
There was once a bug where the Eclipse autoformatter didn't use the proper alignment. I've also submitted code that was formatted wrongly this way by the autoformatter. For a nicely formatted view of a Javadoc you can hover over it.